docs/catalogue/catalogue.rst

```diff
@@ -0,0 +1,99 @@
+
+This catalogue contains references and short descriptions for a number of real examples of pipelines used by labs to generate NWB files, many of which are now published on DANDI!
+
+If it sounds as if any of the projects in this catalogue might be similar to your own experiments, it might be worth checking out how they implemented things and asking questions by raising Issues on the respective repositories.
+
+..note: It is likely that each project pinned itself to a specific minor version of NeuroConv, or its predecessor, nwb-conversion-tools (NCT). We have organized each pipeline according to the version used in descending order (newest version first).
+
+
+
+NeuroConv v0.2
+--------------
+
+`Ahrens Lab <https://ahrenslab.org>`_: `ahrens-lab-to-nwb <https://github.com/catalystneuro/ahrens-lab-to-nwb>`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Ahrens lab was interested in publically sharing the imaging data corresponding to the high-impact publication `Mu Y, Bennett DV, Rubinov M, Narayan S, Yang CT, Tanimoto M, Mensh BD, Looger LL, Ahrens MB. Glia Accumulate Evidence that Actions Are Futile and Suppress Unsuccessful Behavior. Cell. 2019 <https://www.cell.com/cell/fulltext/S0092-8674(19)30621-X?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS009286741930621X%3Fshowall%3Dtrue>`_ and hosting the data on the DANDI archive. In this experiment, paralyzed zebrafish were place in a virtual reality environment which systemetically responded to their attempts to swim by either mimicing movement effects within the environment or not (test of futility). While the subject attempts to engage in swimming behaviors, a light-sheet microscope performs whole-brain scans tracking flourescence activity of neurons, glia, or both dependeing on the session.
+
+`Murthy Lab <https://ahrenslab.org/)>`_: `murthy-lab-to-nwb <https://github.com/catalystneuro/murthy-lab-to-nwb>`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Murthy lab studies fly interactions combined with optical imaging techniques. This project specifically focused on an upcoming paper related to mating behaviors. The most complex aspects of this pipeline include multiple subjects, pose estimation through SLEAP, and visual stimulus reconstruction (from the fly's perspective).
+
+`Fee Lab <https://ahrenslab.org/>`_: `fee-lab-to-nwb <https://github.com/catalystneuro/fee-lab-to-nwb>`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Fee lab was interested in sharing data related to several upcoming publications utilizing both optical imaging and extracellular electrophysiology. The work with this lab focused largely on encoding and visualization of complex heirarchical audio signals which are compared directly to temporally aligned neural activity.
```

Comment:
```suggestion
The Fee lab was interested in sharing data related to several upcoming publications utilizing both optical imaging and extracellular electrophysiology. The work with this lab focused largely on encoding and visualization of complex hierarchical audio signals which are compared directly to temporally aligned neural activity.
```
Otherwise this is a great summary, thanks! Should the extensions (`ndx-sound`, `ndx-hierarchical-behavior-data`) be mentioned here though? Not really `neuroconv` related so not sure..